### ISSUE TITLE
Lints my index.html and complains about <!Doctype html>

### ISSUE THREAD

**Issue by lunchboxer** [sveltejs/eslint-plugin-svelte3#38]
I've got this plugin set up and working well in VSCode. It lints svelte files as configured, but the settings in .vscode/settings.json seem to have it also looking at my index.html file. There it complains "Parsing error: Unexpected token <" when it gets to "<!DOCTYPE html>" because eslint doesn't do HTML. How do get it to ignore "*.html" like it normally does and still deal with svelte files which it now considers HTML files?

my settings.json
```
{
  "files.associations": {
    "*.svelte": "html"
  },
  "files.watcherExclude": {
    "public/bundle": true
  },
  "editor.formatOnType": true,
  "javascript.format.insertSpaceBeforeFunctionParenthesis": true,
  "eslint.autoFixOnSave": true,
  "eslint.validate": [
    "javascript",
    "javascriptreact",
    {
      "language": "html",
      "autoFix": true
    }
  ],
  "eslint.enable": true
}
```

svelte 3.6.7
eslint 6.1
eslint-plugin-svelte3 2.7.2

**Comment by Conduitry** [comment]
To the best of my knowledge, you can't. This is a limitation of the VSCode ESLint extension. The files it runs the linter on can only be specified by their language, and not by a glob. You have a few routes you can take:

- Explicitly add `*.html` to your `.eslintignore`
- Install `eslint-plugin-html` so that when HTML files are linted, they're actually handled correctly, by linting their inline scripts
- Install a VSCode extension that provides a Svelte syntax highlighting language, remove the `*.svelte` -> `html` language association, and enable the extension on files with the new Svelte language.

**Comment by MaxMilton** [comment]
Could you please post your ESLint config? I suspect you might not be setting your `overrides` correctly. Have a look at the project readme for an example of how to confgigure things: https://github.com/sveltejs/eslint-plugin-svelte3/blob/master/README.md

**Comment by lunchboxer** [comment]
eslintrc.js

```javascript
module.exports = {
  extends: ['standard', 'plugin:unicorn/recommended'],
  parserOptions: {
    ecmaVersion: 2019,
    sourceType: 'module'
  },
  env: {
    es6: true,
    browser: true
  },
  plugins: ['svelte3'],
  overrides: [
    {
      files: ['**/*.svelte'],
      processor: 'svelte3/svelte3'
    }
  ],
  rules: {
    'unicorn/filename-case': 0,
    'import/first': 0
  },
  env: {
    browser: true
  }
}
```

**Comment by Conduitry** [comment]
Closing, as this doesn't really have anything to do with this plugin, but rather limitations with the VSCode ESLint extension. I outlined what's going on, and some actions that can be taken, in my comment above.
